packages/neuron-wallet/src/services/node.ts

```diff
@@ -138,13 +143,66 @@ class NodeService {
       return false
     } catch (err) {
       logger.info('CKB:\texternal RPC on default uri not detected, starting bundled CKB node.')
-      return startCkbNode().then(()=> true).catch(err => {
-        logger.info('CKB:\tfail to start bundled CKB with error:')
-        logger.error(err)
-        return false
-      })
+      const isReadyToStart = await this.detectDependency()
+      return isReadyToStart ? this.startNode() : this.showGuideDialog()
     }
   }
+
+  private showGuideDialog = () => {
+    const I18N_PATH = `messageBox.ckb-dependency`
+    return dialog.showMessageBox({
+      type: 'info',
+      buttons: ['skip', 'install-and-exit'].map(label => i18n.t(`${I18N_PATH}.buttons.${label}`)),
+      defaultId: 1,
+      title: i18n.t(`${I18N_PATH}.title`),
+      message: i18n.t(`${I18N_PATH}.message`),
+      detail: i18n.t(`${I18N_PATH}.detail`),
+      cancelId: 0,
+      noLink: true,
+    }).then(({ response }) => {
+      switch (response) {
+        case 1: {
+          // open browser and shutdown
+          const VC_REDIST_URL = `https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads`
```

Comment:
Maybe we should put this URL in `env`?

Reply:
> Maybe we should put this URL in `env`?

It seems that the link has nothing to do with the environment so why should it be in the `env`?

Reply:
My initial thought is that it is a URL variable, which is normally not hardcoded and should be referenced from a place responsible for configurations. It is cleaner to update this type of variable in a configuration file, which is easier to plugin to a CI task when necessary.

But I think you are right, the incentive of relocating the value of this variable is not that obvious.